### DISCUSSION TITLE
Song speed shouldn't change in song select

### DISCUSSION THREAD

**Discussion by Saperxd** [ppy/osu#20992] [Ideas]
It doesn't feel good when you're browsing songs, I would rather listen to how it supposed to sound. 

The only time I wouldn't mind for the speed to change is when I'm changing the speed multiplier in the mod settings.

**Comment by nagi-desuuu** [comment]
The reason for this is that you can gauge how fast the beatmap overall is going to pan out.
This was missing in stable and has caused confusion for a lot of players who thought they had Double Time or Half Time disabled, when the latter is in fact active.
Essentially, what I'm saying is that **THE SPEED CHANGE IS THERE SO THAT IT CAN REMIND YOU THAT SPEED-CHANGING MODS ARE ENABLED!!!**

**Comment by Saperxd** [comment]
So imagine this. You're playing the game and the only thing you do is play DT. The entire time you're not playing you have to listen to songs at twice the speed. Try playing for 2 hours straight like that. It's not feasible. At least an option to disable it should be available.

**Reply by mk56-spn** [reply]
"You're playing the game and the only thing you do is play DT"

right so most of that time, like 90% of it unless you spend all day in song select, will be in game play where youll be listening to the song DTed whether you want to or not.

if it bothers you so much to spend that remaining fraction of time listening to the songs with DT then why not take the half a second itll take you to click f1  then F  and be on your merry way listening to the song without DT, this seems like asking for a solution to a non issue

**Reply by Saperxd** [reply]
If I play 500 maps per session that would add up to 8 minutes and 20 seconds wasted on disabling and enabling mods. Very fun :)
Thanks for the advice tho!
